index.bs

```diff
@@ -3442,21 +3444,25 @@ To <dfn>collect nodes using accessibility attributes</dfn> with given |context n
 
   1. Let |match| be true.
 
-  1. If |selector| [=map/contains=] "<code>role</code>":
+  1. If |context node| implements {{Element}}:
```

Comment:
Should it be HTMLElement here too?

Reply:
No, I think this works for any kind of Element.